### PR TITLE
myWidget Ad extension

### LINKED ISSUE #6965: Intent to implement: mywidget for amp-ad
# Overview
Provides support for amp-ad type=mywidget
It will be widget with content recommendations.

Project site https://widget.my.com/

MyWidget is a service for personalizing content and increasing user metrics.
Our NeuroClick technology uses behavioral data of eighty million users, collaborative filtering algorithms and neural networks. NeuroClick delivers the most relevant content to every user of the site.
myWidget is a free platform used by web publishers to improve metrics and audience loyalty.

We have amp-iframe implementation now.
example for mobile https://www.google.com.ua/amp/s/hi-tech.mail.ru/amp/news/iphone-6-specs-official/

It's can look like horizontal or vertical block
amp_example

# Usage
Component will have data-cid (campaign id) attribute
```
<amp-embed height=300
    type=mywidget
    data-cid=campaign_id>
</amp-embed>
```

### PR DESCRIPTION
**Pull request by 3lvcz**
Pull-request for "myWidget" ad.

Closes #6965

(broken closed one #7252)

### CODE REVIEW

**Comment by lannka** [comment]
@3lvcz pls fix travis

**Comment by 3lvcz** [comment]
I fixed previous build error (missed property annotation), but it fails on tests now. It's our code is broken?

**Comment by jridgewell** [comment]
Restarted the test build. Please check back in a bit.

File: ads/mywidget.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/mywidget.js]: nit: 2017

File: ads/mywidget.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/mywidget.md]: nit: 2017

File: ads/mywidget.js (hunk 1 above)
Comment (zhouyx) [inline ads/mywidget.js]: nit: 2017

File: ads/mywidget.md (hunk 2 above)
Comment (zhouyx) [inline ads/mywidget.md]: 2017

File: ads/mywidget.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      type="mywidget"
+      data-cid="your_campaign_id">
+  <div placeholder>Loading myWidget recommendations.</div>
+  <div fallback>No recommendations for you.</div>
+</amp-embed>
+```
+
+Supported parameters:
+
+- `data-cid`
```
Comment (zhouyx) [inline ads/mywidget.md]: I see that `data-cid` is a required parameter,  I would suggest indicate that it is required here.

**Comment by zhouyx** [comment]
Few comments. LGTM in general

**Comment by 3lvcz** [comment]
Fix year in license headers and describe `cid` as required attribute in mywidget.md

**Comment by zhouyx** [comment]
@3lvcz Thanks for the Pull Request. Merged.

**Pull request merged**